app/cdap/components/WrangleHome/Components/WrangleCard/__test__/WrangleCard.test.tsx

```diff
@@ -17,37 +17,58 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import WrangleCard from '../index';
+import * as reducers from 'components/Connections/Create/reducer';
```

Comment:
don't need * as

Reply:
asterisks is needed as we are importing the file as a module and using the functions that are exposed for mocking the function